### PR TITLE
Feature/hydration dom

### PR DESCRIPTION
**Pull request by dreardon1021**
#### What's this PR do?
Adds the ability for the hydration data to be added to the DOM
#### Where should the reviewer start?
The script.js function populateHydrationData()
#### How should this be manually tested?
Check to make sure data is populating correctly
#### Questions:
-We may need to change layout to a horizontal column if we need to display more info
for other categories?

### CODE REVIEW

File: src/scripts.js [hunk 1]
```diff
@@ -8,20 +9,28 @@ const userStepGoal = document.querySelector('.user-step-goal');
 const userGoalAverage = document.querySelector('.step-goal-average');
 const friendsList = document.querySelector('.friends-list');
 const stepComparison = document.querySelector('.step-goal-comparison')
+const dayHydrationBox = document.querySelector('.hydration-day')
+const weekHydradtionBox = document.querySelector('.hydration-week')
 
-function populateUserInfo(id) {
-  let user = new User(userRepo.getUserData(id))
+function populateUserInfo() {
+  let user = new User(userRepo.getUserData(42));
```
Comment (jonathan-tschida) [inline src/scripts.js]: Why did you get rid of the id parameter?

File: src/scripts.js [hunk 2]
```diff
@@ -8,20 +9,28 @@ const userStepGoal = document.querySelector('.user-step-goal');
...
-  friendsList.innerHTML = friendsList.innerHTML = user.friends.map
-  (friendID => userRepo.getUserData(friendID).name)
+  friendsList.innerHTML = friendsList.innerHTML = user.friends.map(friendID => userRepo.getUserData(friendID).name)
 }
 
-populateUserInfo(42);
\ No newline at end of file
+function populateHydrationInfo(id, date) {
+  dayHydrationBox.innerHTML = hydration.getDay(id, date);
+  let weekObj = hydration.getWeek(id, date);
```
Comment (jonathan-tschida) [inline src/scripts.js]: weekData might be a more symantic name, this is an array, not an object anyway

File: src/scripts.js [hunk 3]
```diff
@@ -8,20 +9,28 @@ const userStepGoal = document.querySelector('.user-step-goal');
 const userGoalAverage = document.querySelector('.step-goal-average');
 const friendsList = document.querySelector('.friends-list');
 const stepComparison = document.querySelector('.step-goal-comparison')
+const dayHydrationBox = document.querySelector('.hydration-day')
+const weekHydradtionBox = document.querySelector('.hydration-week')
```
Comment (jonathan-tschida) [inline src/scripts.js]: typo: weekHydradtionBox => weekHydrationBox

same typo further down

**Comment by jonathan-tschida** [comment]
Overall the dashboard implementation is kind of weak.  You're putting information on the DOM, but it means nothing to the user.  Let's go ahead and merge this once you fix some minor things, but we need to do some pairing to decide how to represent this info to the user.

**Comment by dreardon1021** [comment]
I figured we would get the info to display to start. Then implement a more semantic info display in each category. Once we can get stuff to display then we can go in there and make the HTML/CSS more organized for better displayed info

File: src/scripts.js (hunk 1 above)
Comment (dreardon1021) [reply src/scripts.js]: Ah was playing around with stuff cuz something was going on. Just forgot to switch back

File: src/scripts.js (hunk 2 above)
Comment (dreardon1021) [reply src/scripts.js]: Switched

File: src/scripts.js (hunk 3 above)
Comment (dreardon1021) [reply src/scripts.js]: Fixed

**Comment by jonathan-tschida** [comment]
I get that, I just would have liked to seen like 7 separate html elements and some sort of forEach that changes the innerText of each

**Pull request merged**